source/includes/_sub_category_search_metrics.md

```diff
@@ -0,0 +1,158 @@
+## Search Metrics for Business Sub Categories
+
+#### Resource Overview
+
+```
+Available Methods:      GET
+URI Format:             /client_reports/sub_category_search_metrics/[gmaid]?[query_params]
+```
+
+#### Usage
+
+Use GET to retrieve search metric stat report for a given advertiser and business sub category id
+
+### Business Category - Campaign Table Associations
+![Business Sub Category Campaign Associations](/images/business_sub_category_associations.png)
+
+#### Parameters
+
+When using the GET method, the results can be filtered using these parameters:
+
+| Param                         | Function                                           | Required   | Default                   |
+| ----------------------------- | -------------------------------------------------- | ---------- | ------------------------- |
+| `campaign_id[]`               | Returns the search metrics for campaign_id list    | `REQUIRED` |                           |
+| `global_master_campaign_id[]` | Restrict results to one or more specific campaigns | `OPTIONAL` | all gmcids for advertiser |
+
+#### Example Local Dev Curls:
+
+###### Retrieve data for a given GMAID and campaign_id (list)
+
+```
+curl -H "Authorization: token reachanalyticsreportingservicetoken"  "localhost:3001/client_reports/sub_category_search_metrics/TEST_1?campaign_id[]=USA_1234&campaign_id[]=USA_5678
+```
+
+Report data is returned as JSON e.g.:
+
+```json
+{
+  "report_type": "sub_category_search_metrics",
+  "report_data": {
+    "campaigns": [
+      {
+        "campaign_id": "USA_1234",
```

Comment:
The campaign prefix is redundant, **campaign_id** can be replaced by **id** and **campaign_name** can be replaced by **name**.  This should be consistent with other API responses.

Reply:
Thanks so much for calling out those. ❤️ the Consistency!!